airesources/Clojure/src/MyBot.clj

```diff
@@ -0,0 +1,56 @@
+(ns MyBot
+  (:require [clojure.java.io :as clj.io]
+            [hlt.networking :as io]
+            [hlt.game-map :refer [*player-id* *map-size* *bot-name*
+                                  *owner-ships* *ships* *planets*]]
+            [hlt.utils :as utils]
+            [hlt.entity :as e]
+            [hlt.navigation :as navigation])
+  (:import (java.io PrintWriter))
+  (:gen-class))
+
+(defmacro with-updated-map
+  [& body]
+  `(let [m# (io/read-map)]
+     (binding [*owner-ships* (:owner-ships m#)
+               *ships* (:ships m#)
+               *planets* (:planets m#)]
+       ~@body)))
+
+(def my-bot-name "mybot") ;; Replace this with your bot name here.
```

Comment:
You sure don't want to give it some personalized default name? :P

Reply:
Well, since you insist :)